executor/common_kvm.h

```diff
@@ -5,6 +5,12 @@
 
 // Common KVM-related definitions.
 
+#define INVALID_GPA ((uintptr_t)-1)
+#define ARCH_EXECUTOR_FN_GPA(f, exec_offset)                                                            \
+	((((uintptr_t)(f) >= (uintptr_t)&__start_guest) && ((uintptr_t)(f) < (uintptr_t)&__stop_guest)) \
+	     ? ((uintptr_t)(f) - (uintptr_t)&__start_guest + (exec_offset))                             \
+	     : (INVALID_GPA))
```

Comment:
This looks like a bad failure mode for a programming error. Can we fail build instead?

Reply:
Done.